tests/source/match.rs

```diff
@@ -100,6 +100,14 @@ fn matches() {
     }
 }
 
+fn match_skip() {
+    let _ = match Some(1) {
+        #[rustfmt_skip]
+        Some(n) => n,
```

Comment:
Could you change both patterns to something non-canonical to check that rustfmt does not change the `Some` line, but does reformat the `None` line?